homeassistant/components/ambient_station/manifest.json

```diff
@@ -7,5 +7,5 @@
   "integration_type": "hub",
   "iot_class": "cloud_push",
   "loggers": ["aioambient"],
-  "requirements": ["aioambient==2023.04.0"]
+  "requirements": ["aioambient==2023.12.0"]
```

Comment:
Please bump in a separate PR

Reply:
Ok. Will do.

Reply:
Will do once I have moved the climate functionality into `aioambient`.

Reply:
Pending [aoiambient pull request](https://github.com/bachya/aioambient/pull/403). 

Do you still want me to bump the version in a separate PR even though the update is a prerequisite for this PR?

Reply:
Yep! I'd rather merge that first before doing this PR.

Reply:
Done